prov/rxm/src/rxm.h

```diff
@@ -1061,9 +1061,9 @@ rxm_rx_buf_alloc(struct rxm_ep *rxm_ep, struct fid_ep *msg_ep, uint8_t repost)
 }
 
 static inline void
-rxm_rx_buf_release(struct rxm_ep *rxm_ep, struct rxm_rx_buf *rx_buf)
+rxm_rx_buf_release(struct rxm_ep *rxm_ep, struct rxm_rx_buf *rx_buf, uint8_t repost)
```

Comment:
I don't understand the motivation for this change.  rx_buf stores repost as part of the structure.  Passing that in separately from what rx_buf has looks like it opens the possibility for these to not match and would make bugs more likely.